alembic/versions/d8fb58840a71_added_anilist_table.py

```diff
@@ -0,0 +1,32 @@
+"""Added Anilist Table
+
+Revision ID: d8fb58840a71
+Revises: 298629fea1bc
+Create Date: 2023-01-03 00:10:01.041199
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'd8fb58840a71'
+down_revision = '298629fea1bc'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('anilist_users',
+    sa.Column('id', sa.Integer(), autoincrement=True, nullable=False),
```

Comment:
Since the id is the discord id, this does not need `autoincrement`. I believe removing that will still work.

Reply:
u have it in urs